os/fs/driver/mtd/m25px/m25px.c

```diff
@@ -176,6 +180,22 @@
 #define M25P_M25P128_PAGE_SHIFT    8	/* Page size 1 << 8 = 256 */
 #define M25P_M25P128_NPAGES        65536
 
+/* W25Q128JV  capcity :
```

Comment:
capacity*

Reply:
Sorry It was a typo. Thanks for pointing out.